### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by AleksMat**
Updates the content of Readme and configures automatic parsing of Readme into documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,20 +8,19 @@
...
+The **sentinelhub** Python package is the official Python interface for [Sentinel Hub services](https://www.sentinel-hub.com/). It supports most of the services described in the [Sentinel Hub documentation](https://docs.sentinel-hub.com/api/latest/) and any type of [satellite data collections](https://docs.sentinel-hub.com/api/latest/data/), including Sentinel, Landsat, MODIS, DEM, and custom collections produced by users.
 
-The package also supports obtaining data from Amazon Web Service. It can either provide data from public bucket with
-Sentinel-2 L1C imagery or requester pays bucket with Sentinel-2 L2A imagery. If specified the downloaded data can be
-stored in ESA .SAFE format (all types of .SAFE format are supported).
+The package is also a collection of basic tools and utilities for working with geospatial and satellite data. It builds on top of well known packages such as `numpy`, `shapely`, `pyproj`, etc. On the other hand, it serves as a core dependency of [`eo-learn`](https://github.com/sentinel-hub/eo-learn) Python package for creating geospatial data-processing workflows.
```
Comment (zigaLuksic) [inline README.md:15]: The package **is also** a collection of basic tools and utilities ...
-> The package **also provides** a collection of basic tools and utilities ...

**On the other hand, it serves as** a core dependency of ...
-> **It is also** a core dependency of ...

File: README.md [hunk 2]
```diff
@@ -8,20 +8,19 @@
...
 
-# Installation
+The main package resources are [GitHub repository](https://github.com/sentinel-hub/sentinelhub-py), [documentation page](https://sentinelhub-py.readthedocs.io/en/latest/) and [Sentinel Hub forum](https://forum.sentinel-hub.com/).
```
Comment (zigaLuksic) [inline README.md:17]: I would also add a comma before the last `and` (oxford comma is a good thing)

File: README.md [hunk 3]
```diff
@@ -8,20 +8,19 @@
...
 
-The package requires a Python version >= 3.6 and installed C/C++ compiler. The package is available on
-the PyPI package manager and can be installed with
+
+## Installation
+
+The package requires a Python version >= 3.6 and installed C/C++ compiler. The package is available at
```
Comment (zigaLuksic) [inline README.md:22]: and installed C/C++ compiler
-> and **an** installed C/C++ compiler

File: README.md [hunk 4]
```diff
@@ -30,85 +29,79 @@ $ pip install sentinelhub --upgrade
...
+To install the package manually, clone the repository and run
+
 ```
-$ python setup.py build
-$ python setup.py install
+$ pip install .
 ```
 
-Before installing `sentinelhub-py` on **Windows** it is recommended to install package `shapely` from
+Before installing `sentinelhub` on **Windows** it is recommended to install package `shapely` from
```
Comment (zigaLuksic) [inline README.md:41]: recommended to install package
-> recommended to install **the** package

File: README.md [hunk 5]
```diff
@@ -30,85 +29,79 @@ $ pip install sentinelhub --upgrade
...
+Additionally, some examples are explained in Sentinel Hub webinar videos:
 
-# Blog posts
+- [Process API in Python](https://www.youtube.com/watch?v=sX3w3Wd3FBw&list=PL46vEE2ks3tn8NGesSFllgJW5MSYRi4od&index=10&t=2220s)
+- [OGC API in Python](https://www.youtube.com/watch?v=CBIlTOl2po4&list=PL46vEE2ks3tn8NGesSFllgJW5MSYRi4od&index=4&t=1766s)
+
+
+## Blog posts
+
+The package has been used in the background of many projects and use cases described at [Sentinel Hub blog](https://medium.com/sentinel-hub). The following blog posts are about the package itself:
```
Comment (zigaLuksic) [inline README.md:92]: The package **has been used in the background of** many projects and use cases described at
-> The package **played a key role in** many projects and use cases described at

File: README.md (hunk 1 above)
Comment (AleksMat) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (AleksMat) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (AleksMat) [reply README.md]: fixed

**Pull request merged**